spec/components/editor_component_spec.rb

```diff
@@ -46,7 +46,7 @@
       render_inline(described_class.new(**kwargs))
     end
     it "renders a default hint text" do
-      expect(page).to have_css("div", class: "govuk-hint", text: "You can copy and paste any information and keep bullet point formatting.")
+      expect(page).to have_css("div", class: "govuk-hint", text: "You can copy and paste bullet points into the text box.")
```

Comment:
Could we please add the same hint text under the personal statement on the jobseeker side?

<img width="723" alt="image" src="https://github.com/DFE-Digital/teaching-vacancies/assets/94541493/1282e8ac-11f6-4498-aa7e-69e979b74a71">

Reply:
hint text now present and correct - but interestingly, when I tried to paste in bullet points, it didn't work!